### DISCUSSION TITLE
The top 10 posts with *one exception* are against the For You tab

### DISCUSSION THREAD

**Discussion by suprstarrd** [community/community#13500] [Other Feature Feedback, Questions, & Ideas]
Sorting by Top: All under this category shows the top 10 posts being against the tab, with the 9th as the exception. The first page is filled with many, many more. **Please listen to us, and give us the option to turn it off.** That is, let us do that without sacrificing the social aspect of GitHub / the planned "private" mode.

**Comment by Futuza** [comment]
This should be opt-in, not opt-out Github.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
